runbotics/runbotics-desktop/src/core/websocket/listeners/process.listener.ts

```diff
@@ -89,8 +90,11 @@ export class ProcessListener {
             }
 
             const { processId, input, ...rest } = data;
-            const process = await orchestratorAxios
-                .get<IProcess>(`/api/processes/${processId}`, { maxRedirects: 0 })
+            const tenantId = this.storageService.getValue('tenantId');
```

Comment:
If there's a risk of overwriting this value by a user, we need a bugfix task in #tobediscussed